src/modules/partition/core/PartitionLayout.cpp

```diff
@@ -239,6 +241,26 @@ PartitionLayout::execute( Device* dev,
         {
             currentPartition->fileSystem().setLabel( part.partLabel );
         }
+        if ( !part.partFeatures.isEmpty() )
+        {
+            for ( const auto &k : part.partFeatures.keys() )
+            {
+                const auto &v = part.partFeatures.value(k);
+
+                switch (v.type())
+                {
+                case QMetaType::Bool: 
+                    currentPartition->fileSystem().addFeature( { k, v.toBool() } );
+		    break;
+                case QMetaType::LongLong:
+                    currentPartition->fileSystem().addFeature( { k, v.toInt() } );
+		    break;
+                case QMetaType::QString:
+                    currentPartition->fileSystem().addFeature( { k, v.toString() } );
+                    break;
```

Comment:
I think this should have a `default:` for all the cases not-handled (possibly logging that the configuration looks broken).

Reply:
I added the default  case; it says:

```
15:49:14 [2]: WARNING: Ignoring feature "casefold" of type QVariant::QVariantList ; requires type QVariant::bool, QVariant::qlonglong or Variant::QString. 
```